### PR TITLE
Feature: Add restore block util and restoring RPC method

### PR DESCRIPTION
**Pull request by Tirka**

### CODE REVIEW

File: evm-utils/evm-block-recovery/src/extensions.rs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+use solana_transaction_status::{ConfirmedBlock, TransactionWithStatusMeta};
+
+#[derive(Debug)]
+pub struct ParsedInstructions {
+    pub instructions: Vec<EvmInstruction>,
+    pub only_trivial_instructions: bool,
+}
+
+impl ParsedInstructions {
+    pub fn from_native_block(native: ConfirmedBlock) -> Self {
```
Comment (gBaGu) [inline evm-utils/evm-block-recovery/src/extensions.rs]: Do we need this method?

Comment (Tirka) [reply evm-utils/evm-block-recovery/src/extensions.rs]: It's a future stub for the other than a simple transaction kinds of instructions

Comment (gBaGu) [reply evm-utils/evm-block-recovery/src/extensions.rs]: I mean it seems to have the same functionality as the NativeBlockExt::parse_instructions()
Why do we need both pf them?

**Review by gBaGu — APPROVED** [review]

**Comment by Tirka** [comment]
All changes merged into https://github.com/velas/velas-chain/pull/314

**Pull request closed without merging**